docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```
@@ -942,3 +942,88 @@ spec:
   purpose: Informational
   tags: compliance, hunting
   contributors: nabilschear
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Windows Firewall, Domain Profile enabled (Windows)
```

```suggestion
  name: Firewall enabled, domain profile (Windows)
```
I suggest updating the naming. This is because we'd like a user who's looking for firewall policies to find the macOS and Windows policy, realize by looking at the name that they're accomplishing similar things, and click both.

At some point, I think I suggested the old naming 🙃